library/swift/src/RetryPolicy.swift

```diff
@@ -44,6 +44,25 @@ public final class RetryPolicy: NSObject {
     self.retryOn = retryOn
     self.perRetryTimeoutMS = perRetryTimeoutMS
   }
+
+  /// Converts the retry policy to a set of headers recognized by Envoy.
+  ///
+  /// - returns: The header representation of the retry policy.
+  func toHeaders() -> [String: String] {
```

Comment:
I usually prefer to keep the conversion methods in a separate entity so the source object is decoupled from the target object

Reply:
Where would you prefer we put these in this case?

Reply:
Talked f2f. Let's move it to another file